src/main/webapp/app/exam/participate/exercises/exercise-overview-page/exam-exercise-overview-page.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, EventEmitter, Input, OnChanges, Output, OnInit } from '@angular/core';
+import { Exercise, ExerciseType, getIcon, getIconTooltip } from 'app/entities/exercise.model';
+import { ExamPageComponent } from 'app/exam/participate/exercises/exam-page.component';
+import { ChangeDetectorRef } from '@angular/core';
+import { StudentExam } from 'app/entities/student-exam.model';
+import { ExamExerciseOverviewItem } from 'app/entities/exam-exercise-overview-item.model';
+import { ExamParticipationService } from 'app/exam/participate/exam-participation.service';
+import { Submission } from 'app/entities/submission.model';
```

Comment:
`'Submission' is defined but never used  `

Reply:
Done